tuned/plugins/plugin_modules.py

```diff
@@ -85,9 +87,14 @@ def _instance_verify_static(self, instance, ignore_missing, devices):
 			mpath = "/sys/module/%s" % module
 			if not os.path.exists(mpath):
 				ret = False
-				log.error(consts.STR_VERIFY_PROFILE_FAIL % "module '%s' is not loaded" % module)
+				emsg = consts.STR_VERIFY_PROFILE_FAIL % "module '%s' is not loaded" % module
+				log.error(emsg)
+				vlog.add_log(instance.name, {module: {"message": emsg, "result": ret}})
 			else:
-				log.info(consts.STR_VERIFY_PROFILE_OK % "module '%s' is loaded" % module)
+				ret = True
```

Comment:
Should this line be here? If something already failed and this will be last run through cycle, it would overwrite False result.